NF-Tools-Schemas/animal-model/submitAnimalModel.json

```
@@ -0,0 +1,190 @@
+      "title": "What is the strain nomenclature?"
+    },
+    "animalModelDisease": {
+      "type": "string",
+      "title": "What is the animal model disease?",
+      "enum": [
+        "Neurofibromatosis type 1",
+        "Neurofibromatosis type 2",
+        "No known disease",
+        "Schwannomatosis"
+      ]
+    },
+    "animalModelManifestation": {
+      "type": "string",
+      "title": "What is the animal model of manifestation?",
+      "enum": [
+        "astrocytoma",
+        "high grade glioma",
+        "malignant peripheral nerve sheath tumor",
+        "optic nerve glioma",
+        "spinal development",
+        "plexiform neurofibroma",
+        "metabolic function",
+        "heart malformation",
+        "neural crest hyperplasia",
+        "cognition",
+        "growth",
+        "acute myeloid leukemia",
+        "no symptoms",
+        "Other"
+      ]
+    },
+    "mutationTypes": {
+      "type": "array",
+      "title": "Do any of the mutation types apply?",
+      "items": {
+        "type": "string",
+        "enum": [
+          "Disruption caused by insertion of vector",
+          "Duplication",
+          "Insertion",
+          "Insertion of gene trap vector",
+          "Intergenic deletion",
+          "Intragenic deletion",
+          "Inversion",
+          "Loss of heterozygosity (unspecified mechanism)",
+          "Nucleotide repeat expansion",
+          "Nucleotide substitutions",
+          "Single point mutation",
+          "Translocation",
+          "Transposon insertion",
+          "Viral insertion",
+          "Other"
+        ]
+      }
+    },
+    "alleleType": {
+      "type": "string",
+      "title": "What is the allele type?",
+      "enum": [
+        "Conditional ready",
+        "Constitutively active",
+        "Dominant negative",
+        "Humanized sequence",
+        "Hypomorph",
+        "Inducible",
+        "Inserted expressed sequence",
+        "Knockdown",
+        "Modified isoform(s)",
+        "No functional change",
+        "Null/knockout",
+        "RMCE-Ready",
+        "Recombinase",
+        "Reporter",
+        "Transactivator",
+        "Transposase",
+        "Transposon concatemer",
+        "Other"
+      ]
+    },
+    "mutationMethod": {
+      "type": "string",
+      "title": "What was the mutation method applied?",
+      "enum": [
+        "Chemically and radiation induced",
+        "Chemically induced (ENU)",
+        "Chemically induced (other)",
+        "Endonuclease-mediated (CRISPR)",
+        "Endonuclease-mediated (TALEN)",
+        "Gene trapped",
+        "QTL",
+        "Radiation induced",
+        "Spontaneous",
+        "Targeted",
+        "Transgenic",
+        "Transposon induced",
+        "Other"
+      ]
+    },
+    "affectedGeneSymbol": {
+      "type": "string",
+      "title": "What is the Affected Gene Symbol?"
+    },
+    "affectedGeneName": {
+      "type": "string",
+      "title": "What is the Affected Gene Name?"
```

```suggestion
      "title": "What is the Affected Gene Name?"
```

We can remove this - there are databases that have this information.